### ISSUE TITLE
swipe to delete

### ISSUE THREAD

**Issue by shubham49** [shubham49/react-pwa#5]
functionality when users swipe on task then it deletes it

**Comment by rohan2734** [comment]
hey i would like to work on this issue, can you mention what  and where you want to delete?

**Comment by shubham49** [comment]
Whenever user swipes right on the task, then it deletes from the list

**Comment by rohan2734** [comment]
ok i would like to implement this and work on this issue, can you assign me ?

**Comment by shubham49** [comment]
You can raise PR against this

**Comment by rohan2734** [comment]
ok but if you can assign me i can start working on it, becuase when some other person may again ask to work on this issue right?

**Comment by rohan2734** [comment]
@shubham49  thanks, and i would raise the PR soon but , please merge it on the dates of october only.

**Comment by rohan2734** [comment]
hey @shubham49  
I have done npm install, but I have this error, what packages do i need to install more?
```
npm ERR! sharp@0.23.4 install: `(node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)`

npm ERR! Failed at the react-pwa@2.2.10 start script.
```

**Comment by shubham49** [comment]
Delete node_modules, package-lock.json and run npm i

**Comment by rohan2734** [comment]
i need to fork the repo and then  commit right?

**Comment by shubham49** [comment]
Yes, fork the repo commit your changes on branch, push them on your fork, then raise PR

**Comment by rohan2734** [comment]
> Delete node_modules, package-lock.json and run npm i

that wouldnt solve i think, what do i need to do for that? are you facing the same issue? do we need to seperately install react-pwa like create-react-app? and what about sharp?

Please help me in the setup, i am still facing the same issue.

**Comment by shubham49** [comment]
@rohan2734 you need to just run npm install, no other package needs to be installed.

**Comment by rohan2734** [comment]
> @rohan2734 you need to just run npm install, no other package needs to be installed.

ya i just did that still the errr exists.

**Comment by shubham49** [comment]
try running `npm cache clean --force` before installing node modules

**Comment by rohan2734** [comment]
@shubham49  it is not because of that, i think i need to install some react-pwa cli.

**Comment by shubham49** [comment]
@rohan2734  you don't need that. everything is in package.json that is required.

**Comment by rohan2734** [comment]
@shubham49  ya but idk why it is showing errors.
